src/ray/raylet/worker_pool.cc

```diff
@@ -402,6 +402,9 @@ std::tuple<Process, StartupToken> WorkerPool::StartWorkerProcess(
 #elif defined(__linux__)
       static const std::string kLibraryPathEnvName = "LD_LIBRARY_PATH";
 #elif defined(_WIN32)
+      // USERPROFILE is used in pathlib.Path.home()
```

Comment:
Does this actually make a difference / what happens if we don't add this? This codepath is only taken for C++ I think, so it shouldn't make a difference for Python :)

Reply:
> Does this actually make a difference

Apparently not, since tests are still failing. I am grasping at straws trying to figure out why. One theory I had was that the environment is not being set up properly for windows. It seems, grepping in the virtualenv I use to run tests, that `USERPROFILE` is expected to exist when running python programs, which inherit the environment from the enclosing worker. If `USERPROFILE` is not set in the environment, it will not be inherited, and things like `pathlib.Path.home()` will fail, which I did see in logs locally.